doc/api/api_changes_3.4/behaviour.rst

```diff
@@ -1,2 +1,15 @@
 Behaviour changes
 -----------------
+
+GTK key name changes
+~~~~~~~~~~~~~~~~~~~~
+
+The handling of non-ascii keypresses (as reported in the KeyEvent passed to
+``key_press_event``-handlers) in the GTK backends now correctly reports unicode
+characters (e.g., €), and respects NumLock on the numpad.
+
+The following key names have changed; the new names are consistent with those
+reported by the Qt backends:
+- The "Break/Pause" key (keysym 0xff13) is now reported as "pause" instead of
+  "break" (this is also consistent with the X key name).
```

Comment:
For some reason sphinx is unhappy here:

> /home/circleci/project/doc/api/api_changes_3.4/behaviour.rst:14: WARNING: Unexpected indentation.
/home/circleci/project/doc/api/api_changes_3.4/behaviour.rst:15: WARNING: Block quote ends without a blank line; unexpected 

Maybe one cannot continue the list item with double quotes? Try moving "of" to this line.

Reply:
I think I just need a blank line before the list starts.